embedded-graphics/src/lib.rs

```diff
@@ -242,7 +242,7 @@
 #![doc(
     html_logo_url = "https://raw.githubusercontent.com/jamwaffles/embedded-graphics/191fe7f8a0fedc713f9722b9dc59208dacadee7e/assets/logo.svg?sanitize=true"
 )]
-#![no_std]
+// #![no_std]
```

Comment:
Just a friendly reminder to uncomment this

Reply:
Thanks! Even with this comment I'm 99% guaranteed to forget lol